src/figma/darkTheme.json

```diff
@@ -13,25 +13,25 @@
       "description": "For secondary neutral backgrounds."
     },
     "default-hover": {
-      "value": "#2d3034",
+      "value": "#313235",
       "type": "color",
       "parent": "Theme Colors/Dark mode",
       "description": "For \"hover\" states that use background-default."
     },
     "default-pressed": {
-      "value": "#363b3f",
+      "value": "#3f4145",
       "type": "color",
       "parent": "Theme Colors/Dark mode",
       "description": "For \"pressed\" states that use background-alternative."
     },
     "alternative-hover": {
-      "value": "#1e2124",
+      "value": "#1f2123",
       "type": "color",
       "parent": "Theme Colors/Dark mode",
       "description": "For \"hover\" states that use background-alternative."
     },
     "alternative-pressed": {
-      "value": "#272b2f",
+      "value": "#2e3033",
```

Comment:
<img width="1497" alt="Screenshot 2024-05-27 at 10 12 20 AM" src="https://github.com/MetaMask/design-tokens/assets/8112138/bff0ec11-5220-4f26-b345-889da84017bb">

Reply:
Reverted back to production values